content/blog/abstract-classes-and-interfaces/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Abstract Classes and Interfaces
+date: "2019-04-17T21:37:03.284Z"
```

Comment:
Could you estimate the date (doesn't have to be anywhere close to accurate) that you actually learned each topic? Otherwise it looks like you learned a lot in one day in April and it might make you seem more inexperienced than you actually are.

Also, I don't think you need anything more than the date bit here.